sample/GoogleIO_17/event

```diff
@@ -4,9 +4,9 @@
     "latitude": 37.426851, 
     "longitude": -122.080768, 
     "location_name": "Shoreline Amphitheatre, Mountain View, CA", 
-    "start_time": "2017-05-17T10:00:00", 
-    "end_time": "2017-05-19T17:00:00", 
-    "timezone": "UTC", 
+    "start_time": "2017-05-17T10:00:00-07:00", 
+    "end_time": "2017-05-19T17:00:00-07:00", 
+    "timezone": "PST", 
```

Comment:
Please update it to `America/Los_Angeles`. It is best practise to use full name instead of abbreviation. Server also returns full timezone id.

refer http://www.world-timedate.com/timezone/timezone.php?city_id=969